docs/Berger_POPC_test_case/comp_JMelcrProg/launch.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Launch buildH to reconstruct traj w Hs
+buildH ../start_128popc.pdb -l Berger_POPC -d ../order_parameter_definitions_MODEL_Berger_POPC.def -x ../popc0-25ns_dt1000.xtc -opx popc0-25ns_dt1000_H.xtc
+
+# Rename files
+mv popc0-25ns_dt1000_H.xtc.pdb start_128popc_H.pdb
+mv popc0-25ns_dt1000_H.xtc.xtc popc0-25ns_dt1000_H.xtc
+
+# Remove buildH OP output
+rm OP_buildH.out
+
+# Get prog from JMelcr on NMRlipids (last version in Python 3)
+wget https://raw.githubusercontent.com/NMRLipids/MATCH/master/scratch/scriptsBYmelcr/calcOrderParametersPYTHON3.py
```

Comment:
Write in comment the date of the last modification of this script:
"Last edit 2017/02/02"

People will know against which version of this script, buildH has been compared.

Reply:
In fact this is not the real date I think. This date corresponds to the last change by Josef when it was in Python 2. Someone moved it to Python 3 but didn't write it there. It doesn't take to much space, again I would keep it here.

Reply:
Oh okay. 
Maybe we can put the url of this file from the last commit of this file : 
https://raw.githubusercontent.com/NMRLipids/MATCH/3d1b8cf2068f41117f651cce69f95926513b5a19/scratch/scriptsBYmelcr/calcOrderParametersPYTHON3.py
Or keep this default url (pointing to master) and explain in the comment that the version downloaded correspond to this precise url (the one above, pointing to this specific commit)
In this way, we keep a trace of the 'correct' script.

I'm not really fan of keeping a "random" script in this manner but it's not a big deal either :)

Reply:
> I'm not really fan of keeping a "random" script in this manner but it's not a big deal either :)

Right, I agree with your point. But to me, it's no random script. I think it's essential  to keep a trace of the real script used to generate the data. If, for some reason, the repo hosting this script is destroyed (OK it's unlikely,  but it can happen sometime), then how can we prove the data were generated properly? Also, if for some reason, we want to regenerate the data, we're stuck.

Reply:
Okay, let's keep it!